modelci/cli/modelhub.py

```
@@ -260,9 +260,13 @@ def convert(
         ),
         register: bool = typer.Option(False, '-r', '--register', is_flag=True, help='register the converted models to modelhub, default false')
 ):
+    model = None
+    if id is None and yaml_file is None:
+        typer.echo("WARNING: Please assign a way to find the target model! details refer to --help")
+        return 0
     if id is not None and yaml_file is not None:
-        typer.echo("Do not use -id and -path at the same time.")
-        typer.Exit()
+        typer.echo("WARNING: Do not use -id and -path at the same time!")
+        return 0
```

You should raise an error rather than just return as normal.